editor/src/components/navigator/navigator.spec.browser2.tsx

```diff
@@ -919,6 +919,105 @@ export default Components
 `,
   })
 
+const projectWithThirdPartyRenderProp = (renderPropSource: string) =>
+  createModifiedProject({
+    [StoryboardFilePath]: `import * as React from 'react'
+import * as Utopia from 'utopia-api'
+import {
+  Storyboard,
+  Scene,
+} from 'utopia-api'
+
+function Card({ header, children }) {
+  return (
+    <div data-uid='root'>
+      <h2>{header}</h2>
+      {children}
+    </div>
+  )
+}
+
+var Playground = ({ style }) => {
+  return (
+    <div style={style} data-uid='dbc'>
+      <Card ${renderPropSource} data-uid='78c'>
+        <p>Card contents</p>
+      </Card>
+    </div>
+  )
+}
+
+export var storyboard = (
+  <Storyboard data-uid='sb'>
+    <Scene
+      style={{
+        width: 521,
+        height: 266,
+        position: 'absolute',
+        left: 554,
+        top: 247,
+        backgroundColor: 'white',
+      }}
+      data-uid='scene'
+      data-testid='scene'
+      commentId='120'
+    >
+      <Playground
+        style={{
+          width: 454,
+          height: 177,
+          position: 'absolute',
+          left: 34,
+          top: 44,
+          backgroundColor: 'white',
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}
+        className='playground'
+        css={{ color: 'red' }}
+        data-uid='pg'
+      />
+    </Scene>
+  </Storyboard>
+)
+`,
+    ['/utopia/components.utopia.js']: `const Components = {
+      '/utopia/storyboard': {
+        Card: {
+          properties: {
+            header: {
+              control: 'jsx',
+              preferredChildComponents: [
+                {
+                  name: 'Heading',
+                  additionalImports: '../src/heading',
```

Comment:
This should be absolute, right? Otherwise it only works in specific files

Reply:
Thanks for catching this, I updated the code so that no imports are added for built-in elements https://github.com/concrete-utopia/utopia/pull/5050/commits/6f71152ea5296bbb221507c86e78878e3b73ed56